web/src/views/repo/pipeline/PipelineWrapper.vue

```diff
@@ -47,14 +47,9 @@
 
       <template #tabActions>
         <div class="flex justify-between gap-x-4 text-wp-text-100 flex-shrink-0 pb-2 md:p-0 mx-auto md:mr-0">
-          <div class="flex space-x-1 items-center flex-shrink-0">
+          <div class="flex space-x-1 items-center flex-shrink-0" :title="created">
             <Icon name="since" />
-            <Tooltip>
-              <span>{{ since }}</span>
-              <template #popper
-                ><span class="font-bold">{{ $t('repo.pipeline.created') }}</span> {{ created }}</template
```

Comment:
are this i18n text is used also somewhere else or can it be removed if not used?